java/examples/src/main/java/com/nedap/retail/services/examples/WorkflowExample.java

```diff
@@ -39,8 +38,8 @@ public static void runExample(final Client client) {
             System.out.println(NEW_LINE + "Capturing some EPCIS events first...");
             final EpcisEventContainer epcisEventsContainer = new EpcisEventContainer();
             epcisEventsContainer.events = createEvents(locationId);
-            MESSAGE_IDS.add(generateEpcisEventID(ORGANIZATION_PREFIX, 1).toString());
-            MESSAGE_IDS.add(generateEpcisEventID(ORGANIZATION_PREFIX, 2).toString());
+            MESSAGE_IDS.add(UUID.randomUUID().toString());
```

Comment:
It should not calculate new random UUID's.
Instead is should use the already calculated id's from `epcisEventsContainer.events`